### ISSUE TITLE
Site compatibility checks/fix

### ISSUE THREAD

**Issue by intika** [intika/Librefox#50]
**Testing compatibility on those sites:**

- https://github.com/jellyfin/
- https://developer.mozilla.org/en-US/docs/Web/Demos_of_open_web_technologies

**Comment by intika** [comment]
Report from a ghack user... 

I cannot access any of the ‘google sites’ in Librefox.

“The error message begins thus:”An error occurred during a connection to google.com. The server uses key pinning (HPKP) but no trusted certificate chain could be constructed that matches the pinset. Key pinning violations cannot be overridden. Error code: MOZILLA_PKIX_ERROR_KEY_PINNING_FAILURE”.

Need help. Thanks

**Comment by Atavic** [comment]
*security.OCSP.enabled* should be set to true to avoid such issues (that may have other causes, BTW).

**Comment by intika** [comment]
@Atavic yes just gathering users feedback but at the end of the day next version will be very soft by default so all of those issues should be fixed in one shoot, but don't worry there will be a menu to enforce the settings

**Comment by elypter** [comment]
>Key pinning violations cannot be overridden.

although you really shouldn't visit a site like this because this quite likely means there is someone doing a man in the middle attack i think users shold still be allowed to do it anyways. after all unencrypted traffic is still allowed in most browsers. it is a good choice to disable it by default like with the nohttp addon but there should be no hardlocks.

**Comment by intika** [comment]
@elypter i know i was willing to ask the user from where he was connecting, but just did not bother to...
